src/components/Menu/Menu/hooks/useSubMenuIndex.tsx

```diff
@@ -6,6 +6,7 @@ export default function useSubMenuIndex() {
 
   const setSubMenuIsOpenByIndex = useCallback(
     (index: number, isOpen: boolean) => {
+      // console.log({ index, isOpen });
```

Comment:
remove

Reply:
sorry i was so excited to send the pr that i forgot to remove comments 😭 😆